tests/runahead/06-release-update.t

```diff
@@ -29,7 +29,9 @@ run_ok $TEST_NAME cylc validate $SUITE_NAME
 TEST_NAME=$TEST_NAME_BASE-check-states
 cylc run $SUITE_NAME
 LOG="$(cylc get-global-config --print-run-dir)/${SUITE_NAME}/log/suite/log"
-poll "! grep -q -F '[bar.2016] status=running: (received)succeeded' '${LOG}' 2>'/dev/null'"
+YYYY="$(date +%Y)"
+poll "! grep -q -F '[bar.${YYYY}] status=running: (received)succeeded' \
+    '${LOG}' 2>'/dev/null'"
```

Comment:
God knows how this one was ever passing - it was specific to 2016!

Reply:
This worked because it used to continue after a failed poll, so effectively the next test runs after a minute or so. Good to discover these flaws in our tests though.